quipucords/fingerprinter/__init__.py

```diff
@@ -755,10 +750,6 @@ def _process_vcenter_fact(source, fact):
     add_fact_to_fingerprint(source, 'vcenter_source', fact,
                             'infrastructure_type', fingerprint,
                             fact_value='virtualized')
-    add_fact_to_fingerprint(source, 'vcenter_source', fact,
```

Comment:
Convert the values here to an infrastructure_type of `bare metal` or `virtualized`

Reply:
The fact that I removed was always set to `True` and the line directly above sets the infrastructure type to `virtualized`.  If the virtualized_is_guest was always `True`, how do I decide between `bare metal` and `virtualized`?